fuel/downloaders/adult.py

```diff
@@ -0,0 +1,24 @@
+from fuel.downloaders.base import default_downloader
+
+
+def fill_subparser(subparser):
+    """Set up a subparser to download the adult dataset file.
+
+    The Adult dataset file `adult.data` and `adult.test` is downloaded from
+    the UCI Machine Learning Repository [UCI].
+
+    .. [UCI] https://archive.ics.uci.edu/ml/datasets/Adult
+
+    Parameters
+    ----------
+    subparser : :class:`argparse.ArgumentParser`
+        Subparser handling the adult command.
+
+    """
+    subparser.set_defaults(
+        func=default_downloader,
+        urls=['https://archive.ics.uci.edu/ml/machine-learning-databases/'
+              'adult/adult.data',
+              'https://archive.ics.uci.edu/ml/machine-learning-databases/'
+              'adult/adult.test'],
+        filenames=['adult.data', 'adult.test'])
```

Comment:
The way `fill_subparser` is implemented has changed slightly. You now need to return the download function instead of setting it as `func`'s default value, i.e.

``` python
def fill_subparser(subparser):
    """[...]"""
    subparser.set_defaults(
        urls=['https://archive.ics.uci.edu/ml/machine-learning-databases/'
              'adult/adult.data',
              'https://archive.ics.uci.edu/ml/machine-learning-databases/'
              'adult/adult.test'],
        filenames=['adult.data', 'adult.test'])
    return default_downloader
```

I forgot to mention it, sorry about that!